frontend/catalyst/compiler.py

```diff
@@ -128,7 +128,6 @@ def run_writing_command(command: List[str], compile_options: Optional[CompileOpt
         "func.func(hlo-legalize-to-linalg)",
         "func.func(mhlo-legalize-to-std)",
         "convert-to-signless",
-        "func.func(scalarize)",
```

Comment:
@erick-xanadu I remember you found that either of two passes was providing similar speedups, one at the hlo level and one at the linalg level. Can we replace this pass with the linalg one (since it no longer exists)?